web-client/src/app/state/bookmark/bookmark.service.ts

```diff
@@ -0,0 +1,99 @@
+import { Injectable } from '@angular/core';
+import { CapacitorHttp } from '@capacitor/core';
+import { ToastController } from '@ionic/angular';
+import { createUrlWith } from 'src/app/utils/http.helpers';
+import { SessionQuery } from '../session.query';
+import { BookmarkStore } from './bookmark.store';
+
+const headers = {
+  'Content-Type': 'application/json',
+  Accept: 'application/json',
+};
+
+@Injectable({ providedIn: 'root' })
+export class BookmarkService {
+  constructor(
+    private bookmarkStore: BookmarkStore,
+    private sessionQuery: SessionQuery,
+    private toastCtrl: ToastController
+  ) {}
+
+  async createBookmark(bookmark: { name: any; address: any }) {
```

Comment:
Could these instances of the `any` type be replaced with `string` instead?

Reply:
@jdvlio done